### ISSUE TITLE
invalid INLINE_ORIGIN lines missing symbol

### ISSUE THREAD

**Issue by willkg** [mozilla/dump_syms#511]
dump_syms is sometimes generating invalid `INLINE_ORIGIN` lines that are missing the symbol and don't parse with symbolic.

```
bad debug file
  caused by: parsing error
  caused by: in section "inline origin record body" at 
"INLINE_ORIGIN 533471"
               ^
,
in section "inline origin id" at 
"INLINE_ORIGIN 533471"
               ^
,
expected whitespace at 
"INLINE_ORIGIN 533471"
                     ^
```

* 1C741F40EDFCF67402C75BE96B8CE5B00 libxul.so
* 597DFDE3A46D01136C46AD51EE54DA080 libxul.so

Both of these were generated with mozilla/dump_syms 2.1.0.

**Comment by willkg** [comment]
I hand-fixed the 1C741F40EDFCF67402C75BE96B8CE5B00 one, but haven't hand-fixed the 597DFDE3A46D01136C46AD51EE54DA080 one, yet. It's possible there are more.

I guess I'll hold off on fixing them and once we fix this and don't need the files around anymore, let me know and I can go fix them.

**Comment by mstange** [comment]
I will take a look.
